### ISSUE TITLE
Show aUST used as collateral in Anchor EARN

### ISSUE THREAD

**Issue by danku-r** [Anchor-Protocol/anchor-web-app#148]
It would be great to show the current value of the aUST holdings on a wallet on Anchor EARN that are used as collateral on Mirror. After posting several charts and videos on Twitter and YouTube, I received a lot of feedback that their aUST is gone. This was connected to the feat, that the aUST would also no appreciated anymore. But, of course, this is a matter of education. Anyway, I personally agree that an overview of the current aUST would be helpful. 

This is probably than connected to an issue of withdrawing on the EARN tab. A figure showing the aUST bonded as collateral and the aUST free for a withdraw would be optimal. Thanks and Best! Danku

**Comment by luigi311** [comment]
Yes having two numbers would be best, a total aUST and an available aUST so users know how much they are able to withdraw and the concept of total and available balance is already common in traditional finance.

**Comment by kevko2020** [comment]
I think this will be difficult because the aUST is literally not in your wallet anymore so it would be hard to track especially if aUST can be taken to multiple other platforms in the future.

**Comment by luigi311** [comment]
Yeah it would be hard now that i think about it because the aUST leaves your wallet. So they would need to track where all your aUST is going and validate that you are actually still entitled to it or if you lost it with a margin call or the protocol took it or lost it.
